sdk/healthinsights/azure-healthinsights-radiologyinsights/azure/healthinsights/radiologyinsights/aio/_operations/_operations.py

```diff
@@ -106,122 +123,128 @@ async def _infer_radiology_insights_initial(
 
     @overload
     async def begin_infer_radiology_insights(
-        self, body: _models.RadiologyInsightsData, *, content_type: str = "application/json", **kwargs: Any
-    ) -> AsyncLROPoller[_models.RadiologyInsightsInferenceResult]:
+        self,
+        id: str,
+        resource: _models.RadiologyInsightsJob,
+        *,
+        expand: Optional[List[str]] = None,
+        content_type: str = "application/json",
+        **kwargs: Any
+    ) -> AsyncLROPoller[_models.RadiologyInsightsJob]:
```

Comment:
Just calling this out again...the naming for this model might not be the best for Python, make sure to discuss this name with python architects.

Reply:
This was changed to RadiologyInsightsInferenceResult